lib/loadEnvs.js

```diff
@@ -3,21 +3,14 @@ const fieldsEnvMap = {
   'baseURL': 'BASE_URL',
   'clientID': 'CLIENT_ID',
   'clientSecret': 'CLIENT_SECRET',
+  'sessionSecret': 'SESSION_SECRET',
 };
 
 module.exports = function(params) {
   Object.keys(fieldsEnvMap).forEach(k => {
-    if (params[k]) {
-      return;
+    if (typeof params[k] === 'undefined') {
+      params[k] = process.env[fieldsEnvMap[k]];
     }
-    params[k] = process.env[fieldsEnvMap[k]];
   });
-
-  if (!params.baseURL &&
```

Comment:
A lot of dancing around to allow a very specific case. Also: sets the URL without TLS.